libcloud/compute/drivers/ec2.py

```diff
@@ -3630,7 +3630,7 @@ def ex_register_image(self, name, description=None, architecture=None,
                           image_location=None, root_device_name=None,
                           block_device_mapping=None, kernel_id=None,
                           ramdisk_id=None, virtualization_type=None,
-                          ena_support=None):
+                          ena_support=None, billing_product=None):
```

Comment:
http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_RegisterImage.html
According to the documentation, I thought it should be singular, just like block device mapping.